docs/tools/qemu.md

```diff
@@ -109,151 +122,173 @@ configuration.
-  | o- fileio ............................................. [Storage Objects: 0]
-  | o- pscsi .............................................. [Storage Objects: 0]
-  | o- ramdisk ............................................ [Storage Objects: 0]
-  | o- user:fbo ........................................... [Storage Objects: 0]
-  | o- user:rbd ........................................... [Storage Objects: 0]
-  | o- user:zbc ........................................... [Storage Objects: 1]
-  |   o- zbc0  [model-HM/zsize-256/conv-10@/var/local/zbc0.raw (20.0GiB) activated]
-  |     o- alua ............................................... [ALUA Groups: 1]
-  |       o- default_tg_pt_gp ................... [ALUA state: Active/optimized]
-  o- iscsi ........................................................ [Targets: 0]
-  o- loopback ..................................................... [Targets: 0]
-  o- vhost ........................................................ [Targets: 1]
-    o- {==naa.5001405a0776dce3==} .......................................... [TPGs: 1]
-      o- tpg1 .............................. [naa.500140533e375d94, no-gen-acls]
-        o- acls ...................................................... [ACLs: 0]
-        o- luns ...................................................... [LUNs: 1]
-          o- lun0 ............................... [user/zbc0 (default_tg_pt_gp)]
-/> exit
-```
-
-The virtual machine can then be started with the emulated ZBC disk attached
-using the World-Wide port name assigned by *targetcli*.
-
-```plaintext
-# qemu-kvm (your options) \
-	-device pcie-root-port,bus=pcie.0,id=rp1 \
-	-device vhost-scsi-pci,wwpn={==naa.5001405a0776dce3==},bus=pcie.0
-```
-
-The disk is listed on the guest with tools such as *lsscsi*.
-
-```plaintext
-# lsscsi -g
-[0:0:0:0]    disk    ATA      QEMU HARDDISK    2.5+  /dev/sda   /dev/sg0
-[6:0:1:0]    zbc     LIO-ORG  TCMU ZBC device  0002  /dev/sdb   /dev/sg1
-```
+*tcmu-runner* can be used to create emulated ZBC host-managed SCSI disks. The
+emulated disk that is created can be used locally on the host using the
+loopback fabric adapter, as explained in the ["Creating an Emulated Disk"
+section of the tcmu-runner page](./tcmu-runner.md#creating-an-emulated-disk).
+
+The emulated ZBC disk can be attached to a vhost virtual adapter for use within
+a KVM guest operating system. This is done in a manner similar to the manner in
+which a physical device is attached. (See ["Attaching a Host Physical
+Disk"](/docs/tools/qemu#attaching-a-host-physical-disk) for the particulars of
+attaching a physical device). 
+
+The following example shows the procedure of illustrates this procedure,
+creating a small 20GB host-managed SCSI disk with 256 MB zones including 10
+conventional zones:
+
+1. **Use targetcli to Create a 20GB Host-managed SCSI Disk**
+
+   Run the following command to create a 20GB host-managed SCSI disk with 256 MB zones (including 10 conventional zones):
```

Comment:
long line

Reply:
I erred on the side of keeping the line together, but nudged by you I have discovered that it is not necessary in order for the ordered list to remain intact.

So: fixed!